### PR TITLE
Implements can_be_injected_by().

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
@PsyCommando pointed out this proc was unused earlier, so I reviewed injection code and added it where I think it was intended to go.

## Why and what will this PR improve
Slightly cleaner code.

## Authorship
@PsyCommando found the issue, I did the code.

## Changelog
Nothing player-facing.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Random pike commit too apparently.

File: code/modules/clothing/masks/smokable.dm [hunk 1]
```diff
@@ -201,6 +201,9 @@
 	ignitermes = "<span class='notice'>USER fiddles with FLAME, and manages to light their NAME.</span>"
 	brand = "\improper Trans-Stellar Duty-free"
 
+/obj/item/clothing/mask/smokable/cigarette/can_be_injected_by(var/atom/injector)
+	return TRUE
```
Comment (out-of-phaze) [inline code/modules/clothing/masks/smokable.dm:205]: Uh, why should this differ from `/atom/movable/can_be_injected_by(var/atom/injector)`? Now this returns true regardless of adjacency or free space. (Free space is checked in the one place `can_be_injected_by()` is called, so if it's unnecessary in the `/atom/movable` override, it should probably be removed from there.)

File: code/modules/mob/living/simple_animal/familiars/familiars.dm [hunk 2]
```diff
@@ -45,6 +45,8 @@
 	icon = 'icons/mob/simple_animal/spaceshark.dmi'
 	pixel_x = -16
 
+	offset_overhead_text_x = 16
```
Comment (out-of-phaze) [inline code/modules/mob/living/simple_animal/familiars/familiars.dm:48]: I assume you didn't mean to include the pike commit?

**Review by out-of-phaze — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/familiars/familiars.dm (hunk 2 above)
Comment (MistakeNot4892) [reply code/modules/mob/living/simple_animal/familiars/familiars.dm:48]: I didn't but it's also one line so I don't really care :U

Comment (out-of-phaze) [reply code/modules/mob/living/simple_animal/familiars/familiars.dm:48]: I think when I originally wrote this comment you hadn't mentioned it and then when I posted it in the morning you had. Sorry! Yeah, it's fine.

File: code/modules/clothing/masks/smokable.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/modules/clothing/masks/smokable.dm:205]: Did some work on that, should be okay now.

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**